doc_src/interactive.rst

```diff
@@ -550,7 +550,7 @@ Custom bindings
 In addition to the standard bindings listed here, you can also define your own with :doc:`bind <cmds/bind>`::
 
   # Just clear the commandline on control-c
-  bind ctrl-c 'commandline -r ""'
+  bind ctrl-c clear-commandline
```

Comment:
yeah I think this is equivalent but since it's the default we should probably use `cancel-commandline` as example here